src/App.js

```diff
@@ -1,5 +1,133 @@
+const MissionUtils = require("@woowacourse/mission-utils");
+
 class App {
-  play() {}
+  play() {
+    MissionUtils.Console.print("숫자 야구 게임을 시작합니다.");
+    const computer = this.computerInput();
+    this.gameManager(computer);
+  }
+
+  computerInput() {
+    const computer = [];
+    while (computer.length < 3) {
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if (!computer.includes(number))
+        computer.push(number);
+    }
+    return computer;
+  }
+
+  gameManager(computer) {
+    MissionUtils.Console.readLine('숫자를 입력해주세요 : ', (player) => {
+      this.playerCheck(player);
+      let [strikeCount, ballCount] = this.checkBallCounts(computer, player.split("").map(Number));
+      this.printBallCounts(strikeCount, ballCount);
+      
+      if(!this.isClear(strikeCount))
+        this.gameManager(computer);
+      else
+        this.retry();
```

Comment:
else 대신 return을 활용해 봐도 좋을 것 같아요!
```suggestion
      if(!this.isClear(strikeCount))
        return this.gameManager(computer);
      this.retry();
```